docs/modules/ROOT/pages/how-to/day2ops/disable-access.adoc

```diff
@@ -0,0 +1,41 @@
+= Disable access to {product}
+
+This guide describes how to remove a user's access to all {zone}s
+
+NOTE: This page will be updated once we've implemented support for selective access to single {zone}s.
+
+== Prerequisites
+
+* Installed {idp} (see xref:appuio-cloud:root:how-to/keycloak-setup.adoc[Install Keycloak])
+* Administrator console access to {idp}
```

Comment:
Does this need more explanation what this actually means having administrator access?

Reply:
Added a day 2 operations how-to explaining where the admin credentials can be found